roles/atomic_upgrade/tasks/main.yml

```diff
@@ -34,10 +34,9 @@
 
 - block:
 
-    - name: Workaround BZ1290281
-      selinux:
-        policy: targeted
-        state: permissive
+    - name: Workaround old atomic command bug: failure b/c /tmp is a symlink
```

Comment:
I guess I'll stand down. I don't know what problem this is intended to fix; it might be different from the one I'm thinking of. It's not worth spending more time on. Sorry for the confusion.